### DISCUSSION TITLE
LIS Learning Material

### DISCUSSION THREAD

**Discussion by AmirhosseinAhrari** [NASA-LIS/LISF#1160] [General]
Hello everyone,
 
I'm very new in LIS and I have no idea how it works. Although a user manual released, it is a little complicated. Accordingly does anyone know any tutorial or learning material on working and running LIS? I will be appreciated if anyone helps. 

Sincerely,

**Comment by LIS-navari** [comment]
Hi Amirhossein, 

Please be more specific. 

Do you have access to discover or not? 
https://github.com/NASA-LIS/LISF/tree/master/docs/discover_quick_start

Were you able to clone the LIS code? 
https://github.com/NASA-LIS/LISF/tree/master/docs/working_with_github

Were you able to generate the LIS, LDT, and LVT executables? 
https://github.com/NASA-LIS/LISF/tree/master/docs/LISF_installation_guide

Were you able successfully run the test cases (tutorials)? 
https://github.com/NASA-LIS/LISF/tree/master/docs/public_testcase_walkthrough

LIS is a big platform with different capabilities. You need to patiently read the documentation and follow the instruction to successfully run the public test cases and eventually your test case. 
 
Thanks
